### PR TITLE
Reverting back default style

### PR DESCRIPTION
**Pull request by agenius-logesh**
https://3.basecamp.com/3775257/buckets/16956142/todos/4270615472

### CODE REVIEW

File: tests/testthat/sample_app/program/global.R [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # -- IMPORTS --
 library(canvasXpress)
 
+
```
Comment (agenius-mohammed-ali) [inline tests/testthat/sample_app/program/global.R:15]: why you need this extra line?

File: tests/testthat/sample_app_both_sidebar/program/global.R [hunk 2]
```diff
@@ -12,6 +12,7 @@
 # -- IMPORTS --
 library(canvasXpress)
 
+
```
Comment (agenius-mohammed-ali) [inline tests/testthat/sample_app_both_sidebar/program/global.R:15]: new line?

File: tests/testthat/sample_app_no_sidebar/program/global.R [hunk 3]
```diff
@@ -12,6 +12,7 @@
 # -- IMPORTS --
 library(canvasXpress)
 
+
```
Comment (agenius-mohammed-ali) [inline tests/testthat/sample_app_no_sidebar/program/global.R:15]: the extra

File: tests/testthat/sample_app_no_sidebar_no_resetbutton/program/global.R [hunk 4]
```diff
@@ -12,6 +12,7 @@
 # -- IMPORTS --
 library(canvasXpress)
 
+
```
Comment (agenius-mohammed-ali) [inline tests/testthat/sample_app_no_sidebar_no_resetbutton/program/global.R:15]: the extra line?

File: tests/testthat/sample_app_r_sidebar/program/global.R [hunk 5]
```diff
@@ -12,6 +12,7 @@
 # -- IMPORTS --
 library(canvasXpress)
 
+
```
Comment (agenius-mohammed-ali) [inline tests/testthat/sample_app_r_sidebar/program/global.R:15]: the extra line

**Review by agenius-mohammed-ali — CHANGES_REQUESTED** [review]

**Review by agenius-mohammed-ali — APPROVED** [review]

**Review by cb4ds — APPROVED** [review]

**Pull request merged**